index.html

```diff
@@ -1,33 +1,49 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" type="text/css" href="styles.css" />
-  <title>Portfolio setup and mobile first</title>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <title>Portfolio setup and mobile first</title>
 </head>
+
 <body>
-    <header>
-        <nav class="navbar">
-            <a class=  "box1" href="https://www.github.com/AlejandraKeber">Alejandra</a>
-            <a class="box2" href="#Menu"><img src="./Images/Navbarmenu.png" alt="Menu"></a>
-        </nav>
-    </header>
+    <div class="wrapper">
+        <header>
+            <nav class="navbar">
+                <a class="box1" href="https://www.github.com/AlejandraKeber">Alejandra</a>
+                <a class="box2" href="#Menu"><img src="./Images/Navbarmenu.png" alt="Menu"></a>
+                <ul class="box3">
+                    <li><a href="#Portfolio">Portfolio</a></li>
+                    <li><a href="#About">About</a></li>
+                    <li><a href="#Contact">Contact</a></li>
+                </ul>
+            </nav>
+        </header>
+
         <main>
-            <section>
+            <section class="main">
```

Comment:
Great job @AlejandraKeber 🤝

- [ ] You may also center the contents of your main page so as to reduce the big black gap left at the bottom part of this section.
![Your main](https://user-images.githubusercontent.com/63917725/186830053-809fd75c-98e3-43ad-a76a-37ab8e78820c.PNG)